packages/esm-patient-chart-app/src/visit/visit-form/visit-date-time.component.tsx

```diff
@@ -9,6 +9,16 @@ import { OpenmrsDatePicker, ResponsiveWrapper } from '@openmrs/esm-framework';
 import { convertToDate, type VisitFormData } from './visit-form.resource';
 import styles from './visit-form.scss';
 
+// Helpers to safely compute min/max across optional values
+const minOf = (...values: Array<number | undefined | null>) => {
+  const nums = values.filter((v): v is number => typeof v === 'number' && Number.isFinite(v));
+  return nums.length ? Math.min(...nums) : undefined;
+};
```

Comment:
Nit: missing new line between the functions
```suggestion
};

```